core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/histogram/HistogramChartOpDesc.scala

```diff
@@ -0,0 +1,106 @@
+package edu.uci.ics.texera.workflow.operators.visualization.histogram
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.texera.workflow.operators.visualization.{VisualizationConstants, VisualizationOperator}
+
+class HistogramChartOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+  @JsonProperty(required = false, defaultValue = "Histogram Chart")
+  @JsonSchemaTitle("Title")
+  @JsonPropertyDescription("Add a title to your histogram chart.")
+  var title: String = "Histogram Chart"
+
+  @JsonProperty(value = "value", required = true)
+  @JsonSchemaTitle("Value Column")
+  @JsonPropertyDescription("Column for counting values.")
+  @AutofillAttributeName
+  var value: String = ""
+
+  @JsonProperty(required = false)
+  @JsonSchemaTitle("Color Column")
+  @JsonPropertyDescription("Column for differentiating data by its value.")
+  @AutofillAttributeName
+  var color: String = ""
+
+  @JsonProperty(defaultValue = "false", required = false)
+  @JsonSchemaTitle("Separate Chart?")
+  @JsonPropertyDescription("Separate histogram chart by value.")
```

Comment:
I don't think it is clean to treat `color` and `separateBy` separately, when both are optional fields. You can remove the boolean value `separate` and use only the optional field `separateBy`'s value to decide whether to fill this in the Python code.

Reply:
Oh, Yes. It is not necessary to have `separate` right now. Thank you. I will remove it.